server/src/controllers/user.controllers.ts

```diff
@@ -119,6 +120,18 @@ class UserController {
 			res.status(400).send(error.message);
 		}
 	};
+
+	getTeammates = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
```

Comment:
if I not mistaken we have router.use('/', teamMates) in user routes; also in my HO better use not req.user.id but to add id in params, cause it would be better to reuse

Reply:
With the current approach, we get rid of the need for user ID validation.